clar2wasm/src/standard/standard.wat

```diff
@@ -1770,4 +1813,7 @@
     (export "hash160-int" (func $hash160-int))
     (export "store-i32-be" (func $store-i32-be))
     (export "store-i64-be" (func $store-i64-be))
+    (export "not" (func $not))
+    (export "is-eq-int" (func $is-eq-int))
+    (export "is-eq-uint" (func $is-eq-uint))
 )
```

Comment:
Is it normal that I don't see the export for `$xor` ?

Reply:
Fixed.